landoui/template_helpers.py

```diff
@@ -32,6 +32,30 @@ def is_user_authenticated() -> bool:
     return helpers.is_user_authenticated()
 
 
+@template_helpers.app_template_global()
+def is_treestatus_user() -> bool:
+    if not is_user_authenticated():
+        return False
+
+    try:
+        userinfo = session["userinfo"]
+    except KeyError:
+        return False
+
+    try:
+        groups = userinfo["https://sso.mozilla.com/claim/groups"]
+    except KeyError:
+        return False
```

Comment:
nit: this could be simplified by checking whether the keys you need exist in the dictionary in a single if statement, removing the need for catching exceptions. E.g.,

```python
if "userinfo" not in session or "https://sso.mozilla.com/claim/groups" not in session["userinfo"]
    return False
```

Or you could technically also combine them into one try/except...

```python
try:
    groups = userinfo["session"]["https://sso.mozilla.com/claim/groups"]
except KeyError:
    return False
```

In its current format, splitting it the way you do doesn't provide any benefit.